src/views/DidManage/DidManage.test.tsx

```diff
@@ -14,8 +15,31 @@ jest.mocked(parseDidUri).mockReturnValue({
   fullDid: 'did:kilt:4rrkiRTZgsgxjJDFkLsivqqKTqdUTuxKk3FX3mKFAeMxsR51',
 } as ReturnType<typeof parseDidUri>);
 
+jest.mock('../../utilities/useSwrDataOrThrow/useSwrDataOrThrow');
+
 describe('DidManage', () => {
-  it('should match the snapshot', async () => {
+  it('should match the snapshot without web3name', async () => {
+    jest.mocked(useSwrDataOrThrow).mockReturnValue(undefined);
+    const { container } = render(
+      <MemoryRouter
+        initialEntries={[
+          generatePath(paths.identity.did.manage.start, { address: 'FOO' }),
+        ]}
+      >
+        <Route path={paths.identity.did.manage.start}>
+          <DidManage
+            identity={
+              identities['4sm9oDiYFe22D7Ck2aBy5Y2gzxi2HhmGML98W9ZD2qmsqKCr']
+            }
+          />
+        </Route>
+      </MemoryRouter>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should match the snapshot with web3name', async () => {
+    jest.mocked(useSwrDataOrThrow).mockReturnValue('FancyName');
```

Comment:
Just nitpicking here, but should the mock value also be lowercase?

Reply:
Good point actually! Mock data should not be much different from the real data.